src/data/VariantSelector.ts

```diff
@@ -184,19 +228,21 @@ export function formatVariantDisplayName(
   if (variantQuery) {
     return variantQuery;
   }
+
   const components = [
-    pangoLineage,
-    nextcladePangoLineage ? nextcladePangoLineage + ' (Nextclade)' : undefined,
-    gisaidClade ? gisaidClade + ' (GISAID clade)' : undefined,
-    nextstrainClade ? nextstrainClade + ' (Nextstrain clade)' : undefined,
-    nucMutations?.join(', '),
-    aaMutations?.join(', '),
-    nucInsertions?.join(', '),
-    aaInsertions?.join(', '),
+    pangoLineage?.toUpperCase(),
+    nextcladePangoLineage ? nextcladePangoLineage.toUpperCase() + ' (Nextclade)' : undefined,
+    gisaidClade ? gisaidClade.toUpperCase() + ' (GISAID clade)' : undefined,
+    nextstrainClade ? nextstrainClade.toUpperCase() + ' (Nextstrain clade)' : undefined,
+    nucMutations && nucMutations.map(mutation => normalizeMutationName(mutation)).join(', '),
+    aaMutations && aaMutations.map(mutation => normalizeMutationName(mutation)).join(', '), // normalizeMutationName(mutation)).join(', ')
```

Comment:
```suggestion
    aaMutations && aaMutations.map(mutation => normalizeMutationName(mutation)).join(', '),
```

Please avoid commented-out code!